Tools/Replay/Replay.cpp

```diff
@@ -814,6 +843,10 @@ void Replay::loop()
         return;
     }
 
+    if (check_solution) {
+        report_checks();
+    }
```

Comment:
Can you explain this change? It looks like it's a big difference to what was done before.

Reply:
Bad rebase.  Fixed.